### PR TITLE
Update spoke_vpc acceptance test to include variables for gateway size

### PR DESCRIPTION
**Pull request by iggynopolis**
Included variables: AWS_GW_SIZE, GCP_GW_SIZE, ARM_GW_SIZE

### CODE REVIEW

**Comment by arvindsree** [comment]
Why do we need this? I thought Azure required since it didnt support in all regions. What about the other two?
In general we should minimize variables in acceptance tests

**Comment by iggynopolis** [comment]
For the other two, AWS and GCP, can we set a default value, but still allow the user to set the variable if they want?

**Comment by arvindsree** [comment]
Yeah..I was going to suggest that

File: aviatrix/resource_spoke_vpc_test.go [hunk 1]
```diff
@@ -33,6 +33,15 @@ func TestAccAviatrixSpokeGw_basic(t *testing.T) {
 
 	preGatewayCheck(t, msgCommon)
```
Comment (arvindsree) [inline aviatrix/resource_spoke_vpc_test.go]: Might need to create one more function to throw error if required variables are not set

**Pull request merged**